jjb/dynamic/scale-ci-pipeline.yml

```diff
@@ -3,7 +3,7 @@
     description: 'Collection of Jenkins jobs to setup osp, ocp and run various scale
       tests on OpenShift. '
     disabled: false
-    name: SCALE-CI-PIPELINE
+    name: scale-ci-pipeline-test
```

Comment:
We need to keep the name as is, scale-ci-pipeline-test is the name of the test job.